src/moscot/base/output.py

```diff
@@ -168,6 +171,98 @@ def chain(self, outputs: Iterable["BaseSolverOutput"], scale_by_marginals: bool
 
         return op
 
+    def sparsify(
+        self,
+        mode: Literal["threshold", "percentile", "min_row"],
+        value: Optional[float] = None,
+        batch_size: int = 1024,
+        n_samples: Optional[int] = None,
+        seed: Optional[int] = None,
+    ) -> "MatrixSolverOutput":
+        """
+        Sparsify the :attr:`transport_matrix`.
+
+        This function sets all entries of the transport matrix below `threshold` to 0 and
+        returns an instance of :class:`moscot.base.MatrixSolverOutput` with the the
+        sparsified transport matrix stored as a :class:`~scipy.sparse.csr_matrix`.
+
+        Parameters
+        ----------
+        mode
+            How to determine the value below which entries are set to 0:
+
+                - 'threshold' - threshold below which entries are set to 0.0.
+                - 'percentile' - determine threshold by percentile below which entries are set to 0.0. Hence, between 0
+                  and 100.
+                - 'min_row' - choose the threshold such that each row has at least one non-zero entry.
+
+        value
+            Value to use for sparsification depending on ``mode``:
+
+                - `threshold` - `value` sets the threshold below which entries are set to 0.0.
```

Comment:
Can you also add the `'`, as \`'threshold'\`? I wanna signify this is a string, not some internal constant.